lib/mayaHydra/flowViewportAPIExamples/footPrintNode/mhFootPrintNode.cpp

```diff
@@ -612,13 +624,20 @@ void MhFootPrint::addedToModelCb()
     auto pickHandler = std::make_shared<FootPrintPickHandler>(obj);
     TF_AXIOM(MayaHydra::PickHandlerRegistry::Instance().Register(_pathPrefix, pickHandler));
 
-    // No need for a path mapper: the parts of the footprint are not selectable
-    // individually, only the Maya shape, so the built-in Maya path mapper does
-    // the job of path mapping for the footprint node.
+    // Register a path mapper to map application UFE paths to scene index paths,
```

Comment:
Remove reliance on DataProducersNodeHashCodeToSdfPathRegistry, use path mapping instead, as is done in FlowViewportAPILocator.